<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="theTableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>721</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="select_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>570</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SelectExperiment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NewExperiment</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
